pyproject.toml

```diff
@@ -57,6 +57,7 @@ xxhash = "3.4.*"
 pyjwt = { version = "2.7.*", extras = ["crypto"] }
 ecdsa = "0.18.*"
 distro = "^1.9.0"
+validators = "^0.34.0"
```

Comment:
Can you confirm if productization is aware of this new dependency and we are using the desired version constraint?

Reply:
Good point.  I meant to check but forgot.  Anyhow, I reached out to PDE and added this request: [AAP-35017](https://issues.redhat.com/browse/AAP-35017)